Cue/app/common/SwitchTableRow.js

```diff
@@ -0,0 +1,51 @@
+// @flow
+
+import React from 'react'
+import { Platform, Switch, Text } from 'react-native'
+
+import CueColors from './CueColors'
+
+import TableRow from './TableRow'
+
+
+const styles = {
+  container: {
+    flex: 1,
+    flexDirection: 'row',
+    alignItems: 'center',
+    justifyContent: 'space-between',
+    paddingVertical: Platform.OS === 'android' ? 16 : undefined,
+  },
+  text: {
+    color: CueColors.primaryText,
+    fontSize: Platform.OS === 'android' ? 16 : 17,
+  },
+}
+
+
+export default class SwitchTableRow extends React.Component {
+  props: {
+    text: string,
+    value: boolean,
+    onPress?: (value: boolean) => void,
+  }
+
+  render() {
+    return (
+      <TableRow
+        style={styles.container}
+        onPress={() => this.props.onPress(!this.props.value)}
```

Comment:
Since `onPress` is optional, Flow will require you do do a check to make sure it's not undefined before calling it, i.e., `() => this.props.onPress && this.props.onPress(!this.props.value)`.